core/test/connection.test.ts

```diff
@@ -1,13 +1,17 @@
 import { Hero, Helpers } from '@ulixee/hero-testing';
-import { Log } from '@ulixee/commons/lib/Logger';
+import Log from '@ulixee/commons/lib/Logger';
 import DefaultBrowserEmulator from '@ulixee/default-browser-emulator';
 import { DependenciesMissingError } from '@ulixee/chrome-app/lib/DependenciesMissingError';
 import ChromeApp from '@ulixee/chrome-app/index';
 import BrowserEngine from '@ulixee/default-browser-emulator/lib/BrowserEngine';
 import Core from '..';
 
 const validate = jest.spyOn(BrowserEngine.prototype, 'verifyLaunchable');
-const logError = jest.spyOn(Log.prototype, 'error');
+// TODO figure out why monorepo broke logging magic, and test if it doesnt
+// break external logic. (Probably due to the way we change proto dynamically
+// and import order now changed
```

Comment:
@blakebyrnes probably due to one importing ts and one js. Do we leave it like this and figure this out later when we continue refactor of repo?

Reply:
Isn't it impossible for it to actually import the ts though? Seems like only way I can see this happened is multiple instances of commons

Reply:
As discussed in private this will be handled in other PR